src/components/data-management/LaunchAnalysisButton.jsx

```diff
@@ -0,0 +1,196 @@
+import React, { useState, useCallback, useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Button, Tooltip, Popconfirm } from 'antd';
+import { useRouter } from 'next/router';
+import moment from 'moment';
+import { updateExperimentInfo } from '../../redux/actions/experimentSettings';
+import {
+  updateProject,
+} from '../../redux/actions/projects';
+
+import fileUploadSpecifications from '../../utils/upload/fileUploadSpecifications';
+import UploadStatus from '../../utils/upload/UploadStatus';
+import pipelineStatus from '../../utils/pipelineStatusValues';
+import integrationTestConstants from '../../utils/integrationTestConstants';
+import generateGem2sParamsHash from '../../utils/data-management/generateGem2sParamsHash';
+import { runGem2s } from '../../redux/actions/pipeline';
+import { updateExperiment } from '../../redux/actions/experiments';
+
+const LaunchButtonTemplate = (props) => {
+  // eslint-disable-next-line react/prop-types
+  const { onClick, disabled, text } = props;
+
+  return (
+    <Button
+      data-test-id={integrationTestConstants.ids.LAUNCH_ANALYSIS_BUTTON}
+      type='primary'
+      disabled={disabled}
+      onClick={onClick}
+    >
+      {text}
+    </Button>
+  );
+};
+
+const LaunchAnalysisButton = () => {
+  const dispatch = useDispatch();
+  const router = useRouter();
+
+  const experiments = useSelector((state) => state.experiments);
+  const samples = useSelector((state) => state.samples);
+  const backendStatus = useSelector((state) => state.backendStatus);
+
+  const projects = useSelector((state) => state.projects);
+  const { activeProjectUuid } = projects.meta;
+  const activeProject = projects[activeProjectUuid];
+
+  const [gem2sRerunStatus, setGem2sRerunStatus] = useState({ rerun: true, hash: null, reasons: [] });
+
+  const launchAnalysis = () => {
+    const activeExperimentId = activeProject.experiments[0];
```

Comment:
Can you not use one single experimentId variable across this file instead of defining a new one here and on line 96?  Also that way `calculateGem2sRerunStatus` won't need an experimentId argument.